tests/test_bedrock_client.py

```
@@ -0,0 +1,191 @@
+    with patch.dict("sys.modules", {"docker": docker_mod}):
+        with pytest.raises(RuntimeError):
+            client.send(["list"])
+    docker_client.close.assert_called_once()
+
+
+# ---------------------------------------------------------------------------
+# send_and_read
+# ---------------------------------------------------------------------------
+
+def test_send_and_read_returns_log_output(client: BedrockClient) -> None:
+    docker_mod, _, container = _fake_docker(b"The time is 6000\n")
```

Ruff (0.16.1)

[warning] Unpacked variable `container` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)